### PR TITLE
Initial Draft

### PR DESCRIPTION
**Pull request by compiledpanda**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,222 @@
...
+  </style>
+</head>
+<body>
+  <header>
+    <h1>John M. Clark</h1>
+    <h4>801-369-1421 - socrates37@gmail.com - Eagle Mountain, Utah, USA</h4>
+  </header>
+  <main>
+    <section id="introduction">
+      <p>Breaks down problems and constructs solutions to solve business needs in a efficient, secure, and maintainable fashion.</p>
```
Comment (thuey) [inline index.html]: `an` efficient

File: index.html [hunk 2]
```diff
@@ -0,0 +1,222 @@
...
+    </section>
+    <section id="skills">
+      <div id="skill-charts">
+        <img src="./skills.svg"/>
```
Comment (thuey) [inline index.html]: I love this visualization

File: index.html [hunk 3]
```diff
@@ -0,0 +1,222 @@
...
+        <img src="./languages.png" style="width: 50%;" alt="Javascript, Java, AWS, HTML/CSS, Docker, Linux, Kubernetes, SQL, NoSQL, Web Components, Go, Php, Jenkins, React, Google Cloud, Bash, Angular"/>
```
Comment (thuey) [inline index.html]: Consider Javascript -> JavaScript

File: index.html [hunk 4]
```diff
@@ -0,0 +1,222 @@
...
+            <li>Dealt with apps at scale from both a data size and concurrency/performance aspect</li>
+            <li>Managed and used both SQL and no-SQL databases extensively</li>
+          </ul>
+        </div>
+        <div class="skill-box">
+          <h2>Frontend</h2>
+          <ul>
+            <li>Comfortable with a range of frameworks and tools (React, Redux, jQuery, Angular)</li>
+            <li>Can maintain and extend legacy code and older/homegrown frameworks</li>
+            <li>Well versed in frontend javascript fundamentals and comfortable building entire applications in vanilla javascript</li>
```
Comment (thuey) [inline index.html]: Well versed -> Well-versed
javascript -> JavaScript

File: index.html [hunk 5]
```diff
@@ -0,0 +1,222 @@
...
+      <div id="skill-charts">
+        <img src="./skills.svg"/>
+        <img src="./languages.png" style="width: 50%;" alt="Javascript, Java, AWS, HTML/CSS, Docker, Linux, Kubernetes, SQL, NoSQL, Web Components, Go, Php, Jenkins, React, Google Cloud, Bash, Angular"/>
+      </div>
+      <div id="skill-boxes">
+        <div class="skill-box">
+          <h2>Backend</h2>
+          <ul>
+            <li>Fluent in a large array of languages and patterns, and able to quickly dive in and pickup a new code base</li>
+            <li>Dealt with apps at scale from both a data size and concurrency/performance aspect</li>
```
Comment (thuey) [inline index.html]: This sentence sounds a little weak. Consider something like "Experience scaling distributed and high volume systems"

File: index.html [hunk 6]
```diff
@@ -0,0 +1,222 @@
...
+            <li>Configured a wide range of application deployments (static, stateless, and stateful) in a number of different ways (Kubernetes, AWS Services, Nomad, VM)</li>
+            <li>Experience monitoring, tracing, and debugging complex infrastructure and micro-service deployments</li>
+          </ul>
+        </div>
+        <div class="skill-box">
+          <h2>Infrastructure</h2>
+          <ul>
+            <li>Strong proponent of declarative infrastructure (Terraform, Cloudformation) but also comfortable with imperative tools</li>
+            <li>Designed, setup, and maintained multiple AWS environments from the VPC up</li>
+            <li>Hooked up several logging and metrics systems to both instrument and diagnose issues (New Relic, Cloudwatch, Prometheus, Elasticsearch)</li>
```
Comment (thuey) [inline index.html]: Hooked up -> Connected

File: index.html [hunk 7]
```diff
@@ -0,0 +1,222 @@
...
+          <ul>
+            <li>Comfortable with a range of frameworks and tools (React, Redux, jQuery, Angular)</li>
+            <li>Can maintain and extend legacy code and older/homegrown frameworks</li>
+            <li>Well versed in frontend javascript fundamentals and comfortable building entire applications in vanilla javascript</li>
+          </ul>
+        </div>
+        <div class="skill-box">
+          <h2>Operations</h2>
+          <ul>
+            <li>Setup and used several CI/CD pipelines (Jenkins, CircleCI, TravisCI)</li>
```
Comment (thuey) [inline index.html]: Setup (noun) -> Set up (verb)

File: index.html [hunk 8]
```diff
@@ -0,0 +1,222 @@
...
+            <li>Experience designing and building a range of software, from high performance APIs to asynchronously coordinated micro-services</li>
+            <li>Understand the tradeoffs between speed to market, maintainability, performance, etc...</li>
+            <li>Comfortable bridging the gap between developers, product, and management to arrive at the best solution for the project</li>
+          </ul>
+        </div>
+        <div class="skill-box">
+          <h2>Leadership</h2>
+          <ul>
+            <li>Experience leading development teams, cross-functional teams, and influencing decisions across multiple teams</li>
+            <li>Able to mentor, advise, and level up developers within an organization</li>
```
Comment (thuey) [inline index.html]: level up -> grow

File: index.html [hunk 9]
```diff
@@ -0,0 +1,222 @@
...
+          </ul>
+        </div>
+      </div>
+    </section>
+    <section id="experience">
+      <h2>Experience</h2>
+      <div class="experience">
+        <h3>Sr. Software Developer at Ancestry<small>Aug 2018 - Aug 2019</small></h3>
+        <ul>
+          <li>Updated and provided security fixes for several older applications</li>
```
Comment (thuey) [inline index.html]: Add a period for consistency

File: index.html [hunk 10]
```diff
@@ -0,0 +1,222 @@
...
+          <li>Maintained Jenkins CI/CD for several teams and providing build/deployment consulting.</li>
```
Comment (thuey) [inline index.html]: providing -> provided

File: index.html [hunk 11]
```diff
@@ -0,0 +1,222 @@
...
+          <li>Redesigned, simplified, and added operational and managerial visibility to the internal GDPR orchestration and compliance application.</li>
+        </ul>
+      </div>
+      <div class="experience">
+        <h3>Chief Architect/Principal Architect at Jolt<small>Dec 2017 - July 2018</small></h3>
+        <ul>
+          <li>Responsible for technical direction, application/infrastructure health, and developer growth.</li>
+          <li>Designed the initial move to an event driven architecture based on Kafka streams.</li>
```
Comment (thuey) [inline index.html]: streams -> Streams

File: index.html [hunk 12]
```diff
@@ -0,0 +1,222 @@
...
+          <li>Designed and laid the foundations for the move to a new GraphQL API.</li>
+        </ul>
+      </div>
+      <div class="experience">
+        <h3>Sr. Software Developer at SolutionStream (Contracted to Qualtrics)<small>Jul - Dec 2017</small></h3>
+        <ul>
+          <li>Shipped a multi-account infrastructure management solution enabling teams to spin-up, configure, and maintain their AWS infrastructure and compute clusters.</li>
+          <li>Allowed developers to securely interact with their infrastructure with a cli utilizing IAM/STS.</li>
+          <li>Wrote terraform modules allowing developers to manage, version, and deploy infrastructure.</li>
```
Comment (thuey) [inline index.html]: terraform -> Terraform
allowing -> , which allowed

File: index.html [hunk 13]
```diff
@@ -0,0 +1,222 @@
...
+        </ul>
+      </div>
+      <div class="experience">
+        <h3>Sr. Software Developer at York Solutions (Contract)<small>Jan - Jul 2017</small></h3>
+        <ul>
+          <li>Provided guidance, experience, and software development to clients.</li>
+          <li>Primary developer for Kindex, a genealogy app providing tools for personal record archives.</li>
```
Comment (thuey) [inline index.html]: providing -> that provides

File: index.html [hunk 14]
```diff
@@ -0,0 +1,222 @@
...
+          <li>Provided full stack development in MySQL, PHP (Phalcon), and Javascript</li>
```
Comment (thuey) [inline index.html]: Javascript -> JavaScript

File: index.html [hunk 15]
```diff
@@ -0,0 +1,222 @@
...
+        </ul>
+      </div>
+      <div class="experience">
+        <h3>Sr. Software Developer at Jolt (Contract)<small>2016 - 2017</small></h3>
+        <ul>
+          <li>Set up CI/CD from Github to Kubernetes, and standardized their Kubernetes deployment.</li>
```
Comment (thuey) [inline index.html]: From what I remember, it was GitLab. In any case, if it was Github, it should be spelled "GitHub".

deployment -> deployments

File: index.html [hunk 16]
```diff
@@ -0,0 +1,222 @@
...
+          <li>Responsible for technical direction, application/infrastructure health, and developer growth.</li>
+          <li>Designed the initial move to an event driven architecture based on Kafka streams.</li>
+          <li>Designed and laid the foundations for the move to a new GraphQL API.</li>
+        </ul>
+      </div>
+      <div class="experience">
+        <h3>Sr. Software Developer at SolutionStream (Contracted to Qualtrics)<small>Jul - Dec 2017</small></h3>
+        <ul>
+          <li>Shipped a multi-account infrastructure management solution enabling teams to spin-up, configure, and maintain their AWS infrastructure and compute clusters.</li>
+          <li>Allowed developers to securely interact with their infrastructure with a cli utilizing IAM/STS.</li>
```
Comment (thuey) [inline index.html]: cli -> CLI

File: index.html [hunk 17]
```diff
@@ -0,0 +1,222 @@
...
+          <li>Primary developer for Kindex, a genealogy app providing tools for personal record archives.</li>
+          <li>Provided full stack development in MySQL, PHP (Phalcon), and Javascript</li>
+        </ul>
+      </div>
+      <div class="experience">
+        <h3>Sr. Software Developer at Jolt (Contract)<small>2016 - 2017</small></h3>
+        <ul>
+          <li>Set up CI/CD from Github to Kubernetes, and standardized their Kubernetes deployment.</li>
+          <li>Created tooling to maintain cohesion between prod, stage, and dev environments.</li>
+          <li>Shipped a cli & gui utility for developers that deployed to any environment with a single click.</li>
```
Comment (thuey) [inline index.html]: cli -> CLI
gui -> GUI

File: index.html [hunk 18]
```diff
@@ -0,0 +1,222 @@
...
+          <li>Designed and prototyped an IoT ingress and processing system for customer data.</li>
+        </ul>
+      </div>
+      <div class="experience">
+        <h3>Personal Project - Trepo/Treehub<small>2015 - 2016</small></h3>
+        <ul>
+          <li>Designed, architected, and implemented a distributable, versioned graph database, a specification for storing genealogical information within that database, a way to search and execute arbitrary graph traversals on the distributed database, and an API to access it all.</li>
```
Comment (thuey) [inline index.html]: Consistently use either "distributable" or "distributed" in this paragraph. I favor "distributed".

"...versioned graph database, including a specification for storing genealogical information, a REST API, and a way to search and execute arbitrary graph traversals."

File: index.html [hunk 19]
```diff
@@ -0,0 +1,222 @@
...
+        </ul>
+      </div>
+      <div class="experience">
+        <h3>Software Engineer at FamilySearch (Contingent)<small>2014 - 2015</small></h3>
+        <ul>
+          <li>Provided customized data and REST APIs for outreach campaigns to FamilySearch users.</li>
+          <li>Wrote a utility that would traverse the FamilySearch Tree and find patron’s pioneer ancestors.</li>
```
Comment (thuey) [inline index.html]: patron's -> patrons'

File: index.html [hunk 20]
```diff
@@ -0,0 +1,222 @@
...
+          <li>Assisted and mentored other frontend developers on the team in Angular & basic HTML/JS.</li>
+        </ul>
+      </div>
+      <div class="experience">
+        <h3>Founder at Genealogy Systems<small>2013 - 2014</small></h3>
+        <ul>
+          <li>Created a tool that let you find what genealogical records existed for a certain place and time by analyzing data we scraped from record providers and querying a custom graph index.</li>
```
Comment (thuey) [inline index.html]: let -> "lets" or maybe "helps"

we scraped -> scraped

File: index.html [hunk 21]
```diff
@@ -0,0 +1,222 @@
...
+          <li>Created FamilySearch tree scanner that showed you things to work on and what to do next.</li>
```
Comment (thuey) [inline index.html]: If it is the name, then "Created FamilySerach Tree Scanner". If it's just a description, then "Created a FamilySearch tree scanner".

you -> users

Consider refining: that showed you things to work on and what to do next -> that generated tasks and described next steps

File: index.html [hunk 22]
```diff
@@ -0,0 +1,222 @@
...
+        </ul>
+      </div>
+      <div class="experience">
+        <h3>CTO and Founder at Find-A-Code, BITS<small>2007 - 2013</small></h3>
+        <ul>
+          <li>Created a subscription based reference website for medical coding and billing professionals.</li>
```
Comment (thuey) [inline index.html]: subscription based -> subscription-based

File: index.html (hunk 15 above)
Comment (compiledpanda) [reply index.html]: Yeah, I think it was GitLab. Thanks!

**Pull request merged**